rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -119,6 +146,15 @@ def test_add_edit_delete_entries(rotkehlchen_api_server: 'APIServer'):
     assert_simple_ok_response(response)
     assert entry.identifier is not None
 
+    # test editing history event
+    assert_editing_works(entries[5], rotkehlchen_api_server, 5)
+    # test editing eth withdrawal event
+    assert_editing_works(entries[6], rotkehlchen_api_server, 6)
+    # test editing eth_block event
+    assert_editing_works(entries[7], rotkehlchen_api_server, 7)
+    # test editing eth_deposit event
+    assert_editing_works(entries[8], rotkehlchen_api_server, 8)
```

Comment:
can you assert before sending the patch request that the event has the type that you expect? Then you can remove the comment